products/green_fast_track/models/product/source/air_quality/source__cats_permit_points.sql

```diff
@@ -3,3 +3,4 @@ SELECT
     variable_id,
     raw_geom
 FROM {{ ref("int_spatial__cats_permits") }}
+WHERE lot_geom IS NULL
```

Comment:
This seems like a consequence of your separating raw_geom and lot_geom - before, when raw_geom has mixed geometry, only points are exported when specified as points.

This might be a bit more legible though.

Reply:
correct, when raw_geom had multiple types, this selection would be via `WHERE ST_GEOMETRYTYPE(raw_geom) = 'ST_MultiPoint'`

but the separate columns change was before these exports and made the actual build logic more legible/traceable too ([`int_spatial__all`](https://github.com/NYCPlanning/data-engineering/blob/dm-fix-gft-exports/products/green_fast_track/models/intermediate/spatial/int_spatial__all.sql#L42))

Reply:
using geometry types (in my experience) is more prone to errors and headaches than using separate columns for these distinct entities (points, lots, polys, buffers)